README.md

```diff
@@ -39,7 +39,7 @@ With clear methods and examples, it aims to speed up development and ensure robu
 
 ## Installation
 
-The News Ladder library is available via Composer. Install it by running:
+The News Ladder SDK is available via Composer. Install it by running:
 
 ```bash
 composer require news-ladder/publisher-api
```

Comment:
Update Require command

Reply:
Done. Have force-pushed the branch 2 times. 😅